src/test/data/JsonSerializableGiatrosBookTest/typicalPatientsGiatrosBook.json

```diff
@@ -5,44 +5,45 @@
     "phone" : "94351253",
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
-    "allergied" : [ "amoxicillin" ]
+    "allergies" : [ "amoxicillin" ]
```

Comment:
Right, I should be cleaning this residue: "allergy", "allergied", "allergies"

Reply:
did it already!